LiteCore/Query/SQLiteFleeceFunctions.cc

```diff
@@ -103,6 +107,77 @@ namespace litecore {
         }
     }
 
+    static void handle_fts_value(const Value* data, alloc_slice& result) {
```

Comment:
`alloc_slice::append` isn't efficient: it calls realloc every time. (Because alloc_slice doesn't remember capacity separately from size.) It'd be better to use a `stringstream`.